packages/help-center/src/hooks/use-opening-coordinates.ts

```diff
@@ -24,6 +24,10 @@ export const calculateOpeningPosition = ( element: HTMLElement ) => {
 		return defaultPosition;
 	}
 
+	// This takes into account the '?' button's padding and align the close button with the help center icon
+	const computedStyle = window.getComputedStyle( element );
```

Comment:
FYI, this call is reallly slow. It will typically force style recalc.

Reply:
Hey 👋  Thank you for the tip. I know it can be relatively slow. Since this happens only when opening help center and is not part of a loop or anything like that, I think this is fine. 

Are you concerned about it? I could try other potential solutions, but I am afraid it might bring more complexity.

Reply:
It happens on every click anywhere on the page when the Help Center is closed :(

It will make every button click slower.

Reply:
I have pushed a new commit that relies on `getComputedStyles` just once, subsequently, it relies on `data-attributes` which is faster than `getComputedStyles`. 

I am happy with this approach, but you are welcome to propose another idea :)

Reply:
That's beautiful! Testing now.